org.ultimatepp.TheIDE.yml

```diff
@@ -0,0 +1,115 @@
+app-id: org.ultimatepp.TheIDE
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.llvm17"]
+command: run-ide.sh
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: upp
+    buildsystem: simple
+    build-commands: 
+      - printf "#define flagFlatpak 1\\n" > uppsrc/uppconfig.h
+      - git apply flatpak.patch
```

Comment:
this is not how you add patches to a build.

Reply:
I just noticed that there is type: patch. I will rework it.